Mathlib/Topology/MetricSpace/Lipschitz.lean

```diff
@@ -670,8 +673,8 @@ theorem LipschitzOnWith.extend_real [PseudoMetricSpace α] {f : α → ℝ} {s :
 
 /-- A function `f : α → (ι → ℝ)` which is `K`-Lipschitz on a subset `s` admits a `K`-Lipschitz
 extension to the whole space.
-TODO: state the same result (with the same proof) for the space `ℓ^∞ (ι, ℝ)` over a possibly
-infinite type `ι`. -/
+The same result (with the same proof) for the space `ℓ^∞ (ι, ℝ)` over a possibly
```

Comment:
Perhaps you can clarify the "with the same proof" clause here? Clearly what you've done is slightly more work, so it's not actually the same.

Reply:
Okay I just deleted the "(with the same proof)" :)